lib/cs_guide_web/controllers/venue_controller.ex

```diff
@@ -375,4 +476,34 @@ defmodule CsGuideWeb.VenueController do
         end
     end
   end
+
+  defp send_email(user, new_user?, venue) do
```

Comment:
@RobStallion could this `send_email` be centralised?
(_I am inclined to agree with a localised `defp` function but just wondering_)